src/ui/server/helpers/mappings/map-application-to-form-fields/index.test.ts

```diff
@@ -52,6 +54,52 @@ describe('server/helpers/mappings/map-application-to-form-fields', () => {
     });
   });
 
+  describe('when an application has exporterBusiness fields', () => {
+    it('should return the relevant exporterBusiness fields as timestamps', () => {
+      const result = mapApplicationToFormFields(mockApplication);
+
+      const expected = {
+        ...mockApplication,
+        [SUBMISSION_DEADLINE]: formatDate(mockApplication[SUBMISSION_DEADLINE]),
+        exporterBusiness: {
+          ...mockApplication.exporterBusiness,
+          [YEARS_EXPORTING]: displayNumberFieldValue(mockApplication.exporterBusiness[YEARS_EXPORTING]),
+          [EMPLOYEES_UK]: displayNumberFieldValue(mockApplication.exporterBusiness[EMPLOYEES_UK]),
+          [EMPLOYEES_INTERNATIONAL]: displayNumberFieldValue(mockApplication.exporterBusiness[EMPLOYEES_INTERNATIONAL]),
+        },
+      };
+
+      expect(result).toEqual(expected);
+    });
+  });
+
+  describe('when an application has exporterBusiness fields set to 0', () => {
```

Comment:
We don't need this test, in here we only care about calling `displayNumberFieldValue`/`transformNumberToString` - those functions have their own tests for the 0 scenario